openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/preprocessing.py

```diff
@@ -282,50 +282,12 @@ def preprocess_parameters(parameters):
 
     # Ajout de paramètres
     # Nouveaux chemins
-    regime_general = parameters.prelevements_sociaux.cotisations_securite_sociale_regime_general
     autres = parameters.prelevements_sociaux.autres_taxes_participations_assises_salaires
-    liberal = parameters.prelevements_sociaux.cotisations_taxes_professions_liberales
-    travail = parameters.marche_travail
-    pss = parameters.prelevements_sociaux.pss
 
     # Valeurs
-    cotsoc.add_child('accident', ParameterNode("accident", data={}))
-    cotsoc.accident.children.update(regime_general.accidents.bareme.children)  # À harmoniser! + Créer params depuis IPP
-    cotsoc.add_child('assiette', ParameterNode("assiette", data={}))
-    cotsoc.assiette.add_child('cantines_titres_restaurants', ParameterNode("cantines_titres_restaurants", data={}))
-    cotsoc.assiette.cantines_titres_restaurants.children.update(regime_general.assiette.cantines_titres_restaurants.children)
-
-    cotsoc.add_child('conge_individuel_formation', ParameterNode("conge_individuel_formation", data={}))
-    cotsoc.conge_individuel_formation.children.update(autres.formation.conge_individuel_formation.children)  # À harmoniser
-
-    cotsoc.add_child('apprentissage_contribution_supplementaire', ParameterNode("apprentissage_contribution_supplementaire", data={}))
-    cotsoc.apprentissage_contribution_supplementaire.children.update(autres.apprentissage.apprentissage_contribution_supplementaire.children)  # À harmoniser
-
-    cotsoc.add_child('gen', ParameterNode("gen", data={}))
-    cotsoc.gen.children['plafond_securite_sociale'] = pss.children['plafond_securite_sociale_mensuel']
-    cotsoc.gen.children['plafond_securite_sociale_annuel'] = pss.children['plafond_securite_sociale_annuel']
-    cotsoc.gen.children['plafond_securite_sociale_horaire'] = pss.children['plafond_securite_sociale_horaire']
-    cotsoc.gen.children['nb_heure_travail_mensuel'] = travail.salaire_minimum.children['nb_heure_travail_mensuel']  # À harmoniser
-    cotsoc.gen.children['smic_h_b'] = travail.salaire_minimum.children['smic_h_b']  # À harmoniser
-
-    cotsoc.add_child('indemnite_fin_contrat', ParameterNode("indemnite_fin_contrat", data={}))
-    cotsoc.indemnite_fin_contrat.children.update(regime_general.indemnite_fin_contrat.children)  # À harmoniser
-
-    cotsoc.add_child('microsocial', ParameterNode("microsocial", data={}))
-    cotsoc.microsocial.children.update(liberal.auto_entrepreneur.children)  # À harmoniser
```

Comment:
Ici je note qu'il faudra harmoniser ces paramètres qu'on a en doublon entre `parameters.prelevements_sociaux.cotisations_taxes_professions_liberales.auto_entrepreneur` et `parameters(period).impot_revenu.rpns.microsocial`

Reply:
@Sasha-Laniece : dois-je faire quelque chose où l'as tu noté quelque part ailleurs ?